zerver/views/streams.py

```diff
@@ -560,6 +560,9 @@ def add_subscriptions_backend(
     can_remove_subscribers_group_id: Annotated[
         Json[int | None], ApiParamConfig("can_remove_subscribers_group")
     ] = None,
+    can_access_stream_topics_group_id: Annotated[
+        Json[int | None], ApiParamConfig("can_access_stream_topics_group")
```

Comment:
I think we had decided to allow the anonymous groups as well here. You can see `can_mention_group` setting used for a user group as a reference.

Reply:
Yes, I updated it. I have a question regarding this: If can_access_stream_topics_group_id is an anonymous group, how will we determine if the stream is a support stream? Currently, we check it with the following condition:
```
can_access_stream_topics_group.named_user_group.name != SystemGroups.EVERYONE

```